client/src/app/fullscreen/download/download-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { DownloadComponent } from './modules/download/download.component';
+
+const routes: Routes = [
+    {
+        path: 'system/media/get/:id',
```

Comment:
I wonder why this works. DO not overwrite system urls. E.g. If I send you a link to a resource and you click on the link, the Haproxy will do not even load the client, since the routing redirects it to the media service.

What are you current urls for the fullscreen projector and the resource downloads? Maybe we can find a nicer download url

Reply:
It works because the full url is `downloads/system/media/get/:id`, I think.

Reply:
Current urls for
- projector: `projector/...`
- resource downloads: `download/...`

Reply:
ah ok, but if it is client controlled, why not just `download/:id`?

Reply:
Because I then can add the whole url of a mediafile to the path. But, I adapted it, now. You can review it.